ui/raidboss/data/05-shb/trial/ruby_weapon-ex.ts

```diff
@@ -375,41 +408,48 @@ export default {
         }
 
         const numAdds = Object.keys(colorToAction).length;
-        let suffix;
-        let actionId;
+        let suffix: string | undefined;
+        let actionId: string | undefined;
```

Comment:
```suggestion
        let suffix: string;
        let actionId: string;
```

Reply:
Mmm, whoops, `actionId` can be undefined here (and then I check it below).

Reply:
Isn't `let variableName: string;` initialized as undefined?

Reply:
Yes, but you can't assign undefined to it.